src/main/java/formflow/library/ScreenController.java

```diff
@@ -86,29 +90,41 @@ public ScreenController(
     log.info("Screen Controller Created!");
   }
 
+  @AllArgsConstructor
+  @Getter
+  private class ScreenConfig {
+    String flowName;
+    String screenName;
+    ScreenNavigationConfiguration screenNavigationConfiguration;
+  }
```

Comment:
If we like this, I can put it out into another class. The problem I'm trying to solve is that you can't get the flow or the screen name from the `screenNavigationConfiguration` object, so I thought I'd do a light weight wrapping of it with that data, for convenience.  If you can think of a better way, let me know. 

It's hard to change the `ScreenNavigationConfiguration` object because it's tightly coupled to the yaml format.